editor/src/components/navigator/left-pane/github-pane/block.tsx

```diff
@@ -78,7 +79,7 @@ export const Block = React.memo((props: BlockProps) => {
             justifyContent: 'flex-start',
             height: '100%',
             display: 'grid',
-            gridTemplateRows: '12px 10px 1fr',
+            gridTemplateRows: '13px 10px 1fr',
```

Comment:
as long as this is the correct length to make the indicator aligned withe the title 👍

Reply:
yeah, it was off by a couple pixels and it was driving me nuts